src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -107,19 +121,20 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
         return TechRecordActions.NONE;
     }
   }
-
-  getVehicleDescription(techRecord: TechRecordModel, vehicleType: VehicleTypes | undefined): string {
-    switch (vehicleType) {
-      case VehicleTypes.TRL:
-        return techRecord.vehicleConfiguration ?? '';
-      case VehicleTypes.PSV:
-        return techRecord.bodyMake && techRecord.bodyModel ? `${techRecord.bodyMake}-${techRecord.bodyModel}` : '';
-      case VehicleTypes.HGV:
-        return techRecord.make && techRecord.model ? `${techRecord.make}-${techRecord.model}` : '';
-      default:
-        return 'Unknown Vehicle Type';
-    }
-  }
+  // This function is never called? Check in PR if ok to remove
+
+  // getVehicleDescription(techRecord: TechRecordModel, vehicleType: VehicleTypes | undefined): string {
```

Comment:
Seems to be called in the vehicle-header component html

Reply:
That's a different function with the same name though? That one is defined in vehicle-header.component.ts

Reply:
Pretty sure it got replaced and forgotten about... I axed it..

Reply:
This should be deleted if it is not used by `VehicleTechnicalRecordComponent` anymore.